libraries/vircadia-client/tests/version.cpp

```diff
@@ -0,0 +1,28 @@
+#include <../src/version.h>
+
+#include <catch2/catch.hpp>
+#include <string>
+
+using namespace std::literals;
+
+TEST_CASE( "Sanity checks for client API version information.", "[client-api-version]" ) {
+
+    auto version = *vircadia_client_version();
+
+    REQUIRE( version.major >= 0 );
+    REQUIRE( version.minor >= 0 );
+    REQUIRE( version.tweak >= 0 );
+    REQUIRE( (version.major != 0 || version.minor != 0 || version.tweak != 0) );
```

Comment:
```suggestion
    REQUIRE(version.major != 0 || version.minor != 0 || version.tweak != 0);
```
Unnecessary parentheses.

Reply:
These are actually required by catch2 in this case, otherwise it complains:
`catch2/catch.hpp:2251:44: error: static assertion failed: chained comparisons are not supported inside assertions, wrap the expression inside parentheses, or decompose it`
I guess has something to do with the use of macros.